src/FacebookEventForwarder.js

```diff
@@ -108,7 +108,12 @@
                         params = cloneEventAttributes(event);
 
                     if (event.CurrencyCode) {
-                        params['currency'] = event.CurrencyCode;
+                        if (event.CurrencyCode === ""){
+                            params['currency'] = 'USD'; // inject USD if CurrencyCode is empty
+                        }
+                        else {
+                            params['currency'] = event.CurrencyCode;
+                        }

```

Comment:
an empty string '' wouldn't get passed the check above it. i would just keep it simple and erase lines 110 through 117 and make it:

```
params['currency'] = event.CurrencyCode || 'USD'
```